test/CadlRanchProjects.Tests/BinaryDataAssert.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
Can we put it in common place to serve all test projects (e.g. cadl-ranch, cadl-ranch-non-azure)? Maybe `AutoRest.TestServer.Tests\Infrastructure`?

it is not good, but if you want to put another common place, we need to update all the test project .csproj file to inclue it.

Reply:
I moved it there